### ISSUE TITLE
Fleksy keyboard - split chars other than space doesn't work

### ISSUE THREAD

**Issue by sosite** [splitwise/TokenAutoComplete#184]
- Android 4.4.2
- Lib: 2.0.2
- [Fleksy keyboard](https://play.google.com/store/apps/details?id=com.syntellia.fleksy.keyboard)

Again this keyboard and their magic. When i press a `.`, `,` or `;` then it automatically add a space after that signs and lib doesn't create a token.

I set split chars as below:

```
char[] splitChar = {' ', ',', ';', '.'};
mTags.setSplitChar(splitChar);
```

Generaly lib has a problem when keyboard do any magic with spans.

Maybe it's simillar issue to https://github.com/splitwise/TokenAutoComplete/issues/108?

**Comment by mgod** [comment]
Thanks for the great bug report! This may be related to not having paste implemented correctly in the token view as well.

**Comment by mgod** [comment]
Hey, I think this was an unrelated bug. Can you update to version 2.0.3 and see if it's still broken for you?

**Comment by sosite** [comment]
It's a little better right now (it works with spaces) but still when I misspell any word (and auto correct try fix them) or press `.`, `,`, `;`  with enabled "Smart Spacing" then still they're no effects.

**Comment by mgod** [comment]
I've got some basic samples working with better detection of Fleksy behavior, but there are too many edge cases for it to be ok to include in the master branch at the moment. This and #183 will need the changes I outlined in #205.

**Comment by mgod** [comment]
Closed in #333.
